core/client/fs/src/main/java/alluxio/client/file/cache/store/LocalPageStore.java

```diff
@@ -159,8 +161,16 @@ public void delete(PageId pageId) throws IOException, PageNotFoundException {
   @VisibleForTesting
   public Path getFilePath(PageId pageId) {
     // TODO(feng): encode fileId with URLEncoder to escape invalid characters for file name
-    return Paths.get(mRoot, Long.toString(mPageSize), getFileBucket(pageId.getFileId()),
-        pageId.getFileId(), Long.toString(pageId.getPageIndex()));
+    return Paths.get(getRoot(pageId).toString(), Long.toString(mPageSize),
+        getFileBucket(pageId.getFileId()), pageId.getFileId(),
+        Long.toString(pageId.getPageIndex()));
+  }
+
+  private Path getRoot(PageId pageId) {
+    // TODO(maobaolong): Refactor to support choose volume policy
+    int index = pageId.hashCode() % mRoots.size();
```

Comment:
I just have a question,  what if we add a new root path?  looks like we're still loading all the pages from the existing root paths, but the index here would be changed,  am I right?
say we already have page1 and page2 in root1, then we add root2, in this case, page2's new index would be on root2, but physically it's still on root1. Would this be an issue?

Reply:
Yeah, it would be an issue if we add a new root for the existing roots for now.

So i decided to do some improvement base on this PR, i can add more choose page policies, something like consistent hash policy, or supply a start rebalance pages during add or remove a root directory.

Reply:
I'm good to have another PR for this, thanks!

Reply:
@maobaolong , any updates on this PR, or will raise another PR to fix this issue?